src/transformers/image_processing_base.py

```diff
@@ -351,6 +368,12 @@ def get_image_processor_dict(
                     revision=revision,
                     subfolder=subfolder,
                 )
+                logger.warning_once(
+                    "You have image processor config saved in `preprocessor.json` file which is deprecated. "
+                    "Image processor configs should be saved in their own `image_preprocessor.json` file. You can rename "
+                    "the file or load and save the processor back which renames it automatically. "
+                    "Loading from `preprocessor.json` will be removed in the future."
+                )
```

Comment:
I remember we discussed it already, I'm do think we will **not** be able to deprecate it with millions of models on the hub using `preprocessor.json`, so I would rather not spam with another warning and keep it as is.